src/ResourceManagement/HDInsightJob/HDInsightJob/packages.config

```diff
@@ -3,10 +3,13 @@
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.DataLake.Store" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net45" />
```

Comment:
Wait a minute, the DataLake client is using a different runtime and a different credenbtial type than the HDInsight client.  This will be a nightmare for customers.  Ideally, you should update this client to AutoRest, or you are forcing your customers to install all of these libraries and create two different credential types.

If this is impossible, you need to provide a way to reuse the credentials fromn the HDInsightManagementClient.  There is an adapter that would allow you to create SubscriptionCloudCredentials from ServiceClientCredcentials here:
```csharp
public class SubscriptionCredentialsAdapter : SubscriptionCloudCredentials
        {
            ServiceClientCredentials _credentials;
            string _subscriptionId;

            public SubscriptionCredentialsAdapter(ServiceClientCredentials wrapped, string subscriptionId)
            {
                _credentials = wrapped;
                _subscriptionId = subscriptionId;
            }
            public override string SubscriptionId
            {
                get { return _subscriptionId; }
            }

            public override Task ProcessHttpRequestAsync(HttpRequestMessage request, CancellationToken cancellationToken)
            {
                return _credentials.ProcessHttpRequestAsync(request, cancellationToken);
            }
        }
```